package.json

```diff
@@ -22,7 +22,8 @@
     "e2e": "playwright test -c ./e2e/config/playwright.config.local.ts",
     "e2e:watch": "npm run e2e -- --ui",
     "e2e:codegen": "playwright codegen",
-    "e2e:screenshots:gen": "./scripts/e2e-gen-screenshots.sh"
+    "e2e:screenshots:gen": "./scripts/e2e-gen-screenshots.sh",
+    "knip": "npx knip --config knip.js"
```

Comment:
The `npx` approach should be okay. We might want to install it as a devDep at some point.